proco/connection_statistics/tests/test_api.py

```diff
@@ -38,3 +38,24 @@ def test_global_stats_queries(self):
                 'get',
                 reverse('connection_statistics:global-stat'),
             )
+
+
+class RealTimeConnectivityApiTestCase(TestAPIViewSetMixin, TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.country_one = CountryFactory()
+        cls.school_one = SchoolFactory(country=cls.country_one, location__country=cls.country_one, geopoint=None)
+
+    def test_create_realtime_api(self):
+        response = self.forced_auth_req(
```

Comment:
please add check for db queries num. it's very important to keep this api optimized